content/parsing.go

```diff
@@ -111,13 +118,20 @@ func commaSeparatedStrToTags(str string) []string {
 	return strings.Split(str, ",")
 }
 
-func timeParse(value string) (time.Time, error) {
+func timeParse(value string) (time.Time, bool, error) {
 	var err error
+	missing := false
+
+	if value == "" {
+		missing = true
+		return time.Time{}, missing, nil
```

Comment:
probably good idea to use named return values (for more than two return values)

Reply:
it rarely makes the code more readable for metime, I only use it when the return values are of the same type, like here in the getActiveStatus(), which was bool, bool, bool.. But let's keep it consistent, it's a good idea